libraries/botbuilder-lg/src/templatesParser.ts

```diff
@@ -380,13 +402,32 @@ export class TemplatesTransformer extends AbstractParseTreeVisitor<any> implemen
     public visitImportDefinition(context: lp.ImportDefinitionContext): any {
         const importStr = context.IMPORT().text;
         const groups = importStr.match(TemplatesParser.importRegex);
-        if (groups && groups.length === 3) {
-            const description = groups[1].trim();
-            const id = groups[2].trim();
-            const sourceRange = new SourceRange(context, this.templates.source);
-            const templateImport = new TemplateImport(description, id, sourceRange);
-            this.templates.imports.push(templateImport);
+        if (!groups || (groups.length !== 3 && groups.length !== 4)) {
+            this.templates.diagnostics.push(this.buildTemplatesDiagnostic(TemplateErrors.importFormatError, context));
+            return;
         }
+
+        const description = groups[1].trim();
```

Comment:
You could refactor this slightly to: 
```typescript
const [, description, id, asAlias] = groups;
...
if (asAlias) {
  ...
}
```

EDIT: May not be as elegant with the `.trim()` piece, so feel free to ignore this suggestion 😄

Reply:
```ts
const [, description, id, asAlias] = groups;
const asAliasArray = asAlias?.split(/\s+/);

const sourceRange = new SourceRange(context, this.templates.source);
const templateImport = new TemplateImport(description, id, sourceRange);

if (asAliasArray?.length !== 2 && asAliasArray[0] !== 'as') {
    this.templates.diagnostics.push(
        this.buildTemplatesDiagnostic(TemplateErrors.importFormatError, context)
    );
    return;
} else {
    templateImport.alias = asAliasArray[1].trim();
}
```

Reply:
Thanks. the Optional chaining is not support in current ts version. (3.5).